bigbluebutton-html5/imports/ui/components/audio/audio-graphql/audio-controls/input-stream-live-selector/buttons/LiveSelection.tsx

```diff
@@ -62,7 +62,7 @@ type DeviceListItemType = {
   iconRight?: string;
   onClick?: Function;
   disabled?: boolean;
-  divider?: true,
+  isSeparator?: boolean;
```

Comment:
As here you are in typescript, you should have two different types, one for the `DeviceListItemType` other for the `Separator`.

/cc @Tainan404